Sources/DeepLearning/Layer.swift

```diff
@@ -293,12 +317,10 @@ public struct AvgPool2D<Scalar: TensorFlowFloatingPoint>: Layer {
 public struct LayerNorm<Scalar: TensorFlowFloatingPoint>: Layer {
     /// The offset value, also known as beta.
     public var offset: Tensor<Scalar>
-
     /// The scale value, also known as gamma.
     public var scale: Tensor<Scalar>
-
+    /// The axis.
```

Comment:
How about slightly more description? e.g. `The axis of normalization.`

Reply:
Will def'ly fix when we have a systematic documentation pass over it.